workspace-loader/karma.conf.js

```diff
@@ -0,0 +1,77 @@
+// Karma configuration
+// Generated on Thu Feb 22 2018 15:39:12 GMT+0200 (EET)
```

Comment:
maybe we could delete this generated line

Reply:
not actual anymore